runtime/src/main/java/org/corfudb/runtime/view/ObjectsView.java

```diff
@@ -164,7 +164,7 @@ public long TXEnd() throws TransactionAbortedException {
         try {
             return TransactionalContext.getCurrentContext().commitTransaction();
         } catch (TransactionAbortedException e) {
-            log.warn("TXEnd[{}] Aborted Exception {}", context, e);
+            log.warn("TXEnd[{}] Aborted Exception ", context, e);
```

Comment:
this log takes two variables but only print one of them?

Reply:
The exceptions don't require a placeholder, internally it just prints it at the end of the message. Previously, we were printing something like this:

2021-09-27T09:36:55.654Z | WARN  |              discovery-service |   o.c.runtime.view.ObjectsView | TXEnd[TX[c489]] Aborted Exception {}
org.corfudb.runtime.exceptions.TransactionAbortedException: ...